12-offer-encoding.md

```diff
@@ -58,6 +58,19 @@ The merchant-pays-user flow (e.g. ATM or refund):
 3. The merchant confirms the *invoice_node_id* to ensure it's about to pay the correct
    person, and makes a payment to the invoice.
 
+The pay-mobile-user flow (e.g. paying a friend back to their mobile node):
+1. The mobile user supplies some always-online node with a static (i.e. 
```

Comment:
How does this relate to the PTLC requirement that was mentioned in the original ML post, is that still a requirement?

Reply:
Yes that was indeed my question. Thanks for the link, understood.